<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="ELW5201" failures="1" tests="7" name="Personal Loan" time="0.063" errors="0" timestamp="2023-01-15T16:40:01 IST">
  <testcase classname="testngdemo.day1" name="asdf" time="0.004"/>
  <testcase classname="testngdemo.day1" name="printother" time="0.001"/>
  <testcase classname="testngdemo.day1" name="printpara" time="0.001"/>
  <testcase classname="testngdemo.day1" name="printt" time="0.001"/>
  <testcase classname="testngdemo.day2" name="printt" time="0.001"/>
  <testcase classname="testngdemo.day2" name="printtestfail" time="0.004">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at testngdemo.day2.printtestfail(day2.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- printtestfail -->
  <testcase classname="testngdemo.day2" name="javaa" time="0.001"/>
  <testcase classname="testngdemo.day2" name="pythonn" time="0.0">
    <ignored/>
  </testcase> <!-- pythonn -->
</testsuite> <!-- Personal Loan -->
